packages/notifications/tests/dummy/app/app.js

```diff
@@ -1,7 +1,7 @@
 import Application from '@ember/application';
 import Resolver from 'ember-resolver';
 import loadInitializers from 'ember-load-initializers';
-import config from './config/environment';
+import config from 'dummy/config/environment';
```

Comment:
`navi-notifications/config/environment`?

Reply:
This came from the upgrade blueprint. It is referencing the dummy app's config file not the addons.